component/dependencies/cloudscale-metrics-collector/component/main.jsonnet

```
@@ -0,0 +1,108 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.cloudscale_metrics_collector;
+
+local namespace = 'appuio-cloud-reporting';
+local image = 'ghcr.io/vshn/cloudscale-metrics-collector:v0.2.0';
+
+
+local labels = {
+  'app.kubernetes.io/name': 'appuio-cloud-reporting',
+  'app.kubernetes.io/managed-by': 'commodore',
+  'app.kubernetes.io/part-of': 'syn',
+};
+
+
+{
+  namespace: {
+    kind: 'Namespace',
+    apiVersion: 'v1',
+    metadata: {
+      name: namespace,
+      labels+: labels,
+    },
+  },
+
+  secret: {
+    assert params.cloudscale.token != null : 'cloudscale.token must be set.',
+    kind: 'Secret',
+    apiVersion: 'v1',
+    metadata: {
+      name: 'cloudscale-api',
+      namespace: namespace,
+      labels+: labels,
+    },
+    stringData: {
+      token: params.cloudscale.token
+    }
+  },
+
+  cronjob: {
+    kind: 'CronJob',
+    apiVersion: 'batch/v1',
+    metadata: {
+      name: 'cloudscale-metrics-collector',
+      namespace: namespace,
+      labels+: labels,
+    },
+    spec: {
+      concurrencyPolicy: 'Forbid',
+      failedJobsHistoryLimit: 768,
```

Why this particular number? I don't think keeping 768 failured jobs around helps anyone :)
```suggestion
      failedJobsHistoryLimit: 5,
```

---

That's what the component-appuio-cloud-reporting uses:
https://github.com/appuio/component-appuio-cloud-reporting/blob/master/tests/golden/defaults/appuio-cloud-reporting/appuio-cloud-reporting/11_backfill.yaml#L14

I don't quite understand where that's coming from though, probably the depths of some jsonnet module. Anyway, I've changed it to 5.